runelite-client/src/main/java/net/runelite/client/plugins/mta/MTAConfig.java

```diff
@@ -27,50 +27,86 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
-@ConfigGroup("mta")
+@ConfigGroup(MTAConfig.GROUP)
 public interface MTAConfig extends Config
 {
+	String GROUP = "mta";
+
+	@ConfigSection(
+			name = "Alchemy Room",
+			description = "",
+			position = 0,
+			closedByDefault = true
+	)
+	String alchemyGroup = "alchemy";
+
+	@ConfigSection(
+			name = "Enchantment Room",
+			description = "",
+			position = 1,
+			closedByDefault = true
+	)
+	String enchantmentGroup = "enchantment";
+
+	@ConfigSection(
+			name = "Graveyard Room",
+			description = "",
+			position = 2,
+			closedByDefault = true
+	)
+	String graveyardGroup = "graveyard";
+
+	@ConfigSection(
+			name = "Telekinetic Room",
+			description = "",
+			position = 3,
+			closedByDefault = true
+	)
+	String telekineticGroup = "telekinetic";
+
+
 	@ConfigItem(
-		keyName = "alchemy",
-		name = "Enable alchemy room",
+		keyName = "alchemyEnable",
+		name = "Enable",
 		description = "Configures whether or not the alchemy room overlay is enabled.",
-		position = 0
+		section = alchemyGroup
 	)
 	default boolean alchemy()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "graveyard",
-		name = "Enable graveyard room",
-		description = "Configures whether or not the graveyard room overlay is enabled.",
-		position = 1
+			keyName = "enchantmentEnable",
```

Comment:
Double check your formatter--these extra tabs are unwanted. `@ConfigItem(` should cause only one extra level of indentation.